### ISSUE TITLE
German de-DE Translation

### ISSUE THREAD

**Issue by coldfire84** [coldfire84/node-red-alexa-home-skill-v3-web#7]
Required for Alexa skill enablement in German-speaking markets.

If you can help with the below please let me know! 

Translation required:

| Skill Text | English Text | 
| ----|----|
|Public Name | Node-RED Smart Home Control |
|One Sentence Description| Add Alexa voice control to your Smart Home Node-RED Flows|
| Detailed Description| Using this bridge you can control a variety of Node-RED endpoints, such as MQTT, HTTP(S), Yamaha AVRs, Chromecast (anything that has a Node-RED integration). This bridge currently supports Power, Brightness, Colour, Colour Temperature, Input, Lock, Thermostat, Playback, Step Speaker and Scene control. You'll need to follow the instructions here to get up and running: https://github.com/coldfire84/node-red-alexa-home-skill-v3-web/wiki/Use-the-Hosted-Instance|
|Example Phrase 1| Alexa, turn on the lights|
|Example Phrase 2| Alexa, pause Lounge TV|
|Example Phrase 3| Alexa, set AV Receiver to HDMI 2|

**Comment by deniscubic** [comment]
Here is my german translation. Hope you can work with it.
Greeting from germany.


Skill Text | English Text | German Text
-- | -- | --
Public Name | Node-RED Smart Home Control | Node-RED Smart Home-Steuerung
One Sentence Description | Add Alexa voice control to your Smart Home Node-RED Flows | Fügt zu Ihren Smart Home-Steuerung Node-RED Flows die Alexa-Sprachsteuerung hinzu
Detailed Description | Using this bridge you can control a variety of Node-RED endpoints, such as MQTT, HTTP(S), Yamaha AVRs, Chromecast (anything that has a Node-RED integration). This bridge currently supports Power, Brightness, Colour, Colour Temperature, Input, Lock, Thermostat, Playback, Step Speaker and Scene control. You'll need to follow the instructions here to get up and running: https://github.com/coldfire84/node-red-alexa-home-skill-v3-web/wiki/Use-the-Hosted-Instance | Mit dieser Bridge können Sie eine Vielzahl von Node-RED-Endpunkten steuern, wie z. B. MQTT, HTTP(S), Yamaha AVRs und Chromecast (alles, was über eine Node-RED-Integration verfügt). Diese Bridge unterstützt derzeit die Einstellungen für An/Aus, Helligkeit, Farbe, Farbtemperatur, Input, Sperre, Thermostat, Wiedergabe, Lautsprecher und Szene. Sie müssen den Anweisungen hier folgen, um loszulegen: https://github.com/coldfire84/node-red-alexa-home-skill-v3-web/wiki/Use-the-Hosted-Instance (Die Dokumentation ist nur in englischer Sprache verfügbar)
Example Phrase 1 | Alexa, turn on the lights | Alexa, schalte das Licht an
Example Phrase 2 | Alexa, pause Lounge TV | Alexa, pausiere lounge Fernseher
Example Phrase 3 | Alexa, set AV Receiver to HDMI 2 | Alexa, schalte AV Receiver auf HDMI 2

**Comment by coldfire84** [comment]
@deniscubic - thank you!

I have setup the skill using this translation, hopefully you can see it now.

**Comment by deniscubic** [comment]
> @deniscubic - thank you!
> 
> I have setup the skill using this translation, hopefully you can see it now.

I can't see the skill in my alexa App. Sorry.

**Comment by Rapejim** [comment]
> I can't see the skill in my alexa App. Sorry.

You can't see it in the Skill Store of Alexa App because it's a beta Skill (for developers). 
But you can see it in "Skills and games - My Skills - Developer"

![screenshot_2019-01-01-22-36-38-845_com amazon dee app](https://user-images.githubusercontent.com/7972545/50576623-e32d2700-0e15-11e9-9166-5c9afe91579c.png)

**Comment by deniscubic** [comment]
Sorry here is my screenshot. My 2 own dev skills. And no beta skill from here.

![img_6137](https://user-images.githubusercontent.com/1503277/50576701-d4e00a80-0e17-11e9-8c09-2e97b545bdd9.PNG)

**Comment by Rapejim** [comment]
> Sorry here is my screenshot. My 2 own dev skills. And no beta skill from here.

But, you are using hosted instance? Or your own instance...

If you are using hosted, did you request the access to skill? (the access is granted by @coldfire84). And activate it on your Alexa App.

**Comment by deniscubic** [comment]
> > Sorry here is my screenshot. My 2 own dev skills. And no beta skill from here.
> 
> But, you are using hosted instance? Or your own instance...
> 
> If you are using hosted, did you request the access to skill? (the access is granted by @coldfire84). And then, you can install/activate it (Skill) on your Alexa App.

I dont understand the question ... what is the difference?

**Comment by Rapejim** [comment]
>  I dont understand the question ... what is the difference?

For use this Skill, you need a web-page (for create and manage the devices), a MongoDB, a MQTT, etc...

If you don't want to create and configure all of this services (it's my case :smile:), you can use the hosted instance.
https://github.com/coldfire84/node-red-alexa-home-skill-v3-web/wiki/Use-the-Hosted-Instance

**Comment by deniscubic** [comment]
> > I dont understand the question ... what is the difference?
> 
> For use this Skill, you need a web-page (for create and manage the devices), a MongoDB, a MQTT, etc...
> 
> If you don't want to create and configure all of this services (it's my case 😄), you can use the hosted instance.
> https://github.com/coldfire84/node-red-alexa-home-skill-v3-web/wiki/Use-the-Hosted-Instance

Yeah i use the hostet thing. But i cant see the skill. No idea why.

**Comment by Rapejim** [comment]
> Yeah i use the hostet thing. But i cant see the skill. No idea why.

Did you follow the instructions of this hosted instance link?
https://github.com/coldfire84/node-red-alexa-home-skill-v3-web/wiki/Use-the-Hosted-Instance
(especially the first step, :wink:)

**Comment by deniscubic** [comment]
> > > I dont understand the question ... what is the difference?
> > 
> > 
> > For use this Skill, you need a web-page (for create and manage the devices), a MongoDB, a MQTT, etc...
> > If you don't want to create and configure all of this services (it's my case 😄), you can use the hosted instance.
> > https://github.com/coldfire84/node-red-alexa-home-skill-v3-web/wiki/Use-the-Hosted-Instance
> 
> Yeah i use the hostet thing. But i cant see the skill. No idea why.

Yes! I ask him! and he told me its done...

**Comment by Rapejim** [comment]
> Yes! I ask him! and he told me its done...

And you must recibe a email from Alexa/Amazon (whit this subject "You're invited to beta test a new Alexa skill") with the link for enable it in Alexa App.

**Comment by coldfire84** [comment]
@deniscubic I've removed and re-added you to the beta test, plus sent a reminder. Hopefully you will get the email this time...

**Comment by coldfire84** [comment]
@deniscubic  I've contacted Amazon support ref this issue.

**Comment by deniscubic** [comment]
> @deniscubic I've removed and re-added you to the beta test, plus sent a reminder. Hopefully you will get the email this time...

No change...

**Comment by deniscubic** [comment]
> @deniscubic I've contacted Amazon support ref this issue.

I think its because of the language. Maybe a englisch skill is not compatible with an german amazon account. Dont know. 
What a shame!

**Comment by coldfire84** [comment]
There is a German Skill available in the market place (that's the translation that you helped with). So this *should* show.

**Comment by deniscubic** [comment]
> There is a German Skill available in the market place (that's the translation that you helped with). So this _should_ show.

Yeah but is it a developer Version? Now open version?

**Comment by coldfire84** [comment]
Yep, it's a developer version. The Spanish version shows, in Spanish, even though the skill is in Development.

**Comment by deniscubic** [comment]
> Yep, it's a developer version. The Spanish version shows, in Spanish, even though the skill is in Development.

Can't see the Skill. No way no idea. ;(

**Comment by coldfire84** [comment]
That's good news, I have added the link I sent you to [the Wiki page](https://github.com/coldfire84/node-red-alexa-home-skill-v3-web/wiki/Use-the-Hosted-Instance) on using the hosted instance.

**Comment by deniscubic** [comment]
Yeah thats cool!

Am So., 6. Jan. 2019 um 20:31 Uhr schrieb coldfire84 <
notifications@github.com>:
